docs/errorhandling.rst

```diff
@@ -59,13 +59,13 @@ you with a function to call on your whim (and in tests).  Note that
 are interchangable when handed to the registration methods or decorator
 (``BadRequest.code == 400``).
 
-You are however not limited to a :exc:`~werkzeug.exceptions.HTTPException`
-or its code but can register a handler for every exception class you like.
+You are however not limited to :exc:`~werkzeug.exceptions.HTTPException`
+or its subclasses but can register a handler for every exception class you like.
```

Comment:
Remove "however".

"code" is correct here, it referrs to HTTP codes.  Something like "You are not limited to HTTPException classes or HTTP codes" would be better.